data_pipeline/twitter/get_old_tweet3_extended.py

```diff
@@ -0,0 +1,335 @@
+import sys
```

Comment:
Is this file itself copied from somewhere, apart from the methods referencing the code in GetOldTweets3?